### ISSUE TITLE
[question] Minecraft shaders?

### ISSUE THREAD

**Issue by nathgit** [markusfisch/ShaderEditor#177]
Is it possible to create Minecraft shaders with this?

**Comment by AntonPieper** [comment]
No, but you can learn GLSL with this app and prototype different effects that you can then copy over to minecraft
